vault/logical_system_paths.go

```diff
@@ -602,6 +602,27 @@ func (b *SystemBackend) sealPaths() []*framework.Path {
 			HelpDescription: strings.TrimSpace(sysHelp["key-status"][1]),
 		},
 
+		{
+			Pattern: "rotate/config$",
+			Fields: map[string]*framework.FieldSchema{
+				"max_operations": &framework.FieldSchema{
+					Type:        framework.TypeInt, //64?
```

Comment:
Well, the kind of numbers we'd need to support are bigger than a signed 32 bit int.  In testing it seems to work fine as an int because we're on 64 bit machines, but there doesn't seem to be a way to ensure 64-bitness in the field system.